gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -135,7 +134,8 @@ final void setManifestExtensions(Map<String, Object> manifestExtensions) {
     final Set<String> getProductDependenciesConfig() {
         // HACKHACK serializable way of representing all dependencies
         return productDependenciesConfig.get().getIncoming().getResolutionResult().getAllComponents().stream()
-                .map(ComponentResult::getId)
+                // intentionally using a lambda as otherwise we break gradle 4.10 support
+                .map(result -> result.getId())
```

Comment:
Have filed https://github.com/iamdanfox/gradle-clippy/issues/3 so we could eventually build a productized solution